ReactiveCocoaTests/KeyValueObservingSpec.swift

```diff
@@ -144,9 +144,31 @@ class KeyValueObservingSpec: QuickSpec {
 				expect(values) == [0, 1, 0, 10]
 			}
 
+			it("should not retain the replaced value") {
+				let parentObject = NestedObservableObject()
+
+				autoreleasepool {
```

Comment:
Since KVO producers retain the object being observed, they were wrapped in autoreleasepool (well, closure does the job too) so that they would deinitialize when out of scope.